dvc/fs/azure.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, repo, config):
 
     def _prepare_credentials(self, config):
         from azure.identity.aio import DefaultAzureCredential
+        # Disable spam from failed cred types for DefaultAzureCredential
+        logging.getLogger("azure.identity.aio").setLevel(logging.ERROR)
```

Comment:
Not sure what the best way to do this is, but DefaultAzureCredential will log XXCredential.get_token failed blah blah loads of times (one for each thread?) for any auth methods it tries which aren't successful before finding one that is

Reply:
What is the level that they are logging those methods, if it is debug then it should be fine.

Reply:
I think warning